youtube_dl/extractor/theintercept.py

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class TheInterceptIE(InfoExtractor):
+    _VALID_URL = r'https://theintercept.com/fieldofvision/(?P<id>.+?)/'
+    _TESTS = [{
+        'url': 'https://theintercept.com/fieldofvision/thisisacoup-episode-four-surrender-or-die/',
+        'info_dict': {
+            'id': 'thisisacoup-episode-four-surrender-or-die',
+            'ext': 'mp4',
+            'title': '#ThisIsACoup – Episode Four: Surrender or Die',
+            'upload_date': '20151218',
+            'description': 'md5:74dd27f0e2fbd50817829f97eaa33140',
+        }
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        video_id = mobj.group('id')
+        webpage = self._download_webpage(url, video_id)
+        title = self._og_search_title(webpage).split(' — ')[0]
+        description = self._og_search_description(webpage)
+        thumbnail = self._og_search_property('image', webpage)
+        content_id = thumbnail.split('/')[-1].split('.')[0]
+        content_url = 'https://content.jwplatform.com/jw6/{content_id}.xml'.format(content_id=content_id)
```

Comment:
there is a new extractor for the jwplatform(you should rebase your work on a newer version).
it can extract all the formats and info.